Utils/azuretools-core/src/com/microsoft/azuretools/core/mvp/model/webapp/AzureWebAppMvpModel.java

```diff
@@ -161,13 +161,7 @@ public IWebApp createWebAppFromSettingModel(@NotNull WebAppSettingModel model) {
 
     // todo: Move duplicated codes to azure common library
     private ResourceGroup getOrCreateResourceGroup(String subscriptionId, String resourceGroup, String region) {
```

Comment:
can this method be removed?

Reply:
This could be removed after we migrate to tasks in app service library, will be done later